testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts

```diff
@@ -1,5 +1,98 @@
 import BaseMethods from "./BaseMethods";
 
+export enum CredentialTypes {
+  UndergraduateCertificate = "undergraduateCertificate",
+  UndergraduateDiploma = "undergraduateDiploma",
+  UndergraduateDegree = "undergraduateDegree",
+  GraduateCertificate = "graduateCertificate",
+  GraduateDiploma = "graduateDiploma",
+  GraduateDegreeOrMasters = "graduateDegreeOrMasters",
+  PostgraduateOrDoctorate = "postGraduateOrDoctorate",
+  QualifyingStudies = "qualifyingStudies",
+}
+
+export enum ProgramLengthOptions {
+  WeeksToLessThanYear = "12WeeksToLessThan1Year",
+  OneToTwoYears = "1YearToLessThan2Years",
+  TwoToThreeYears = "2YearsToLessThan3Years",
+  ThreeToFourYears = "4YearsToLessThan5Years",
+  FiveOrMoreYears = "5YearsOrMore",
+}
+
+export enum RegulatoryBodyOptions {
+  PTIB = "ptib",
+  DQAB = "dqab",
+  ITA = "ita",
+  ICBC = "icbc",
+}
+
+export enum EntranceRequirementsOptions {
```

Comment:
mm..so which filed is this?

Reply:
I believe it represents this 
![image](https://user-images.githubusercontent.com/54600590/208173059-ef5bb665-e824-4282-baea-2ceba5453612.png)

Reply:
Thanks Dheepak!